src/neo/Ledger/StorageItem.cs

```diff
@@ -58,6 +65,45 @@ public void Deserialize(BinaryReader reader)
             IsConstant = reader.ReadBoolean();
         }
 
+        public T[] GetSerializableArray<T>(int max = 0x1000000) where T : ISerializable, new()
+        {
+            if (cached is T[] value) return value;
```

Comment:
I think this should refer to the way of `GetInteroperable<T>()`.

Reply:
But if it's null, it should deserialize the value, instead of create a new entry

Reply:
If it is not `null`, then it should be `T[]`. Otherwise it should throw.

Reply:
Please take a look